user_print.py

```diff
@@ -0,0 +1,22 @@
+
+
+def farewell_greeting_to_user():
+    print("Thank you for choosing to look at some pokemon with us! have a pleasant day")
+
+def ask_user_draw_pokemon():
+    print("Would you like to draw a pokemon card?")
+
+
+def present_pokemon_data(pokemon):
+    print(f"Pokémon ID: {pokemon['id']}")
+    print(f"Name: {pokemon['name'].capitalize()}")
```

Comment:
I don't know if all the Pokémon are single-word named but if there are exceptions then I'd use `.title()`